client/src/app/shared/components/projector/projector.component.ts

```diff
@@ -30,20 +30,17 @@ export const PROJECTOR_CONTENT_FOLLOW: Follow = {
     styleUrls: ['./projector.component.scss']
 })
 export class ProjectorComponent extends BaseComponent implements OnDestroy {
+    private readonly projectorSubject = new BehaviorSubject<ViewProjector | null>(null);
+
     @Input()
     public set projector(projector: ViewProjector) {
-        this.setProjector(projector);
+        this.projectorSubject.next(projector);
```

Comment:
@tsiegleauq Best would be to change this input to an observalbe directly; But this is not directly possible (e.g. the projector list); so this might be done later.